docs/pages/api.js

```diff
@@ -92,15 +91,26 @@ export default class Api extends React.Component {
                         , a compatible renderer written in C++ with bindings for
                         desktop and mobile platforms.
                     </p>
-                    <DrUiNote
-                        title="Pricing for Mapbox GL JS"
-                        imageComponent={<BookImage />}
-                    >
+                    <DrUiNote title="Pricing for Mapbox GL JS" theme="warning">
```

Comment:
In a recent PR we changed this note from the warning theme to the default theme in production: https://docs.mapbox.com/mapbox-gl-js/api/

```suggestion
                    <DrUiNote title="Pricing for Mapbox GL JS">
```